src/tree/FunctionAppTreeItem.ts

```diff
@@ -23,15 +23,15 @@ export class FunctionAppTreeItem implements ILogStreamTreeItem, IAzureParentTree
     private readonly _functionsTreeItem: FunctionsTreeItem;
     private readonly _appSettingsTreeItem: AppSettingsTreeItem;
     private readonly _proxiesTreeItem: ProxiesTreeItem;
-    private readonly _outputChannel: OutputChannel;
+    private readonly _isLinuxPreview: boolean;
 
-    public constructor(client: SiteClient, outputChannel: OutputChannel) {
+    public constructor(client: SiteClient, isLinuxPreview: boolean) {
```

Comment:
I'd rather see 

public constructor(client: SiteClient, options: { isLinuxPreview: Boolean})

That'd be much easier to understand.